### PR TITLE
Consume the leaderboard API

### PR DESCRIPTION
**Pull request by codepantha**
## Changes introduced into the project via this pull request: 👇 
- Submit scores to the API 🌐 
- Fetch all game leaderboard scores from the API and display them in the UI

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -17,13 +17,13 @@ <h2 class="h2">Recent scores</h2>
...
                 <ul class="scores-list">
-                    <li class="score-item ash">John: 100</li>
+                    <!-- <li class="score-item ash">John: 100</li>
                     <li class="score-item">Zoe: 150</li>
                     <li class="score-item ash">Natasha: 55</li>
                     <li class="score-item">Peter: 78</li>
                     <li class="score-item ash">Chloe: 42</li>
                     <li class="score-item">Zerubabel: 190</li>
-                    <li class="score-item ash">Freya: 200</li>
+                    <li class="score-item ash">Freya: 200</li> -->
```
Comment (Dipeshtwis) [inline src/index.html]: - [ ] I would recommend removing the commented code to make the codebase cleaner.

File: src/apiCalls.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { displayMessage, displayScores } from './display';
+
+const gameId = '3L4QvyqAZFTD6eSge8xk';
+
+export async function submitScore(name, score) {
```
Comment (Dipeshtwis) [inline src/apiCalls.js]: - [ ] Ensure the usage of arrow functions instead of the function keyword.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @codepantherr, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [ ] I would recommend updating the screenshot, `live demo link`, and steps to run the project locally to make the readme professional.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (codepantha) [reply src/index.html]: @Dipeshtwis thank you. That was an oversight 🤦‍♂️

File: src/apiCalls.js (hunk 2 above)
Comment (codepantha) [reply src/apiCalls.js]: Done

File: README.md [hunk 3]
```diff
@@ -14,19 +14,26 @@
 
 ## Live Demo 🔗
 
-[Live Demo Link](https://codepantherr.github.io/Portfolio/)
+[Live Demo Link](https://codepantherr.github.io/leaderboard/)
```
Comment (D4nielJ) [inline README.md:17]: [OPTIONAL] If you still haven't deployed I think it would be better to just delete this section of the README file until later.

**Review by D4nielJ — APPROVED** [review]
## Hello @codepantherr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (codepantha) [reply README.md:17]: Okay

**Pull request merged**